borg/locking.py

```diff
@@ -8,17 +8,22 @@
 ADD, REMOVE = 'add', 'remove'
 SHARED, EXCLUSIVE = 'shared', 'exclusive'
 
-# only determine the PID and hostname once.
-# for FUSE mounts, we fork a child process that needs to release
-# the lock made by the parent, so it needs to use the same PID for that.
-_pid = os.getpid()
+# for performance reasons, only determine the hostname once.
+## XXX this sometimes requires live internet access for issuing a DNS query in the background.
+##_hostname = '%s@%s' % (socket.getfqdn(), uuid.getnode())
 _hostname = socket.gethostname()
```

Comment:
12: 1.1 comment
13: 1.1 way
14: 1.0 way

so, keep the 1.0 way or change it to 1.1 behaviour? we do not have stale detection in 1.0.